sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -144,21 +168,19 @@ export interface Span {
     addEvent(name: string, attributes?: Attributes): this;
     addLink(spanContext: SpanContext, attributes?: Attributes): this;
     context(): SpanContext;
-    end(endTime?: number): void;
+    end(endTime?: TimeInput): void;
     isRecordingEvents(): boolean;
     setAttribute(key: string, value: unknown): this;
     setAttributes(attributes: Attributes): this;
     setStatus(status: Status): this;
-    // (undocumented)
-    start(startTime?: number): void;
```

Comment:
Can you point to any discussions in the open telemetry side of the world where it was decided that span will no longer have a "start()"?

Reply:
I'm not sure where the original typings we got were from, but in the git history of opentelemetry-js this method has never existed since the typings file was created.